### PR TITLE
Update app.js

### PR DESCRIPTION
**Pull request by asvsfs**
Bootstrap button's caption didn't show up and onClick event didn't fire , so i passed the event to <a> tag .

### CODE REVIEW

File: examples/jquery-bootstrap/js/app.js [hunk 1]
```diff
@@ -3,10 +3,10 @@
 var BootstrapButton = React.createClass({
   render: function() {
     return (
-      <a {...this.props}
+      <a onClick={this.props.onclick}
```
Comment (chenglou) [inline examples/jquery-bootstrap/js/app.js]: Can you change this to `onClick`?

Comment (asvsfs) [inline examples/jquery-bootstrap/js/app.js]: Well onClick is meant to work for component but it doesnt i thought it prevents confusion maybe

Comment (asvsfs) [inline examples/jquery-bootstrap/js/app.js]: No , onClick doesn't work!

**Comment by chenglou** [comment]
Master seems to work fine (well, with #2884 anyway). I think the bug might be on your side? It's definitely not `onclick/onClick`'s problem as that's only a prop name. If you pass `onClickBlaCustomName` to `BootstrapButton` and it does `onClick={this.props.onClickBlaCustomName}`, you're still fine.

**Comment by asvsfs** [comment]
ah Okay , yes it works that way.

**Comment by chenglou** [comment]
=)

**Pull request closed without merging**